### PR TITLE
Add functionality to delete a certificate

### LINKED ISSUE #674: Allow a cert to be deleted
@SerpentBytes and I are trying to test notifications, and we need a way to create a new cert.  Without manually editing the database, there's no way to delete a certificate right now.

It would be nice if there was a way to delete a certificate for a user.  It wouldn't get used often, but when you need it, this is important.

### PR DESCRIPTION
**Pull request by SerpentBytes**
## Description
Closes #674 by adding functionality to delete certificates. WIP, awaiting reviews. 

### Steps to test
- Run the project locally
- Request a certificate
- Interact with the Delete button:
delete cert sc

### CODE REVIEW

File: app/components/certificate/description.tsx [hunk 1]
```diff
@@ -57,7 +59,24 @@ export default function DescriptionSection({
             <Flex justifyContent="flex-end">
               <Form method="post">
                 <Button type="submit" rightIcon={<RepeatIcon />} isDisabled={!isRenewable}>
-                  Renew Certificate
+                  Renew
+                </Button>
+              </Form>
+              <Form method="post">
+                <input type="hidden" name="intent" value="delete-certificate" />
+                <Button
```
Comment (humphd) [inline app/components/certificate/description.tsx:67]: Let's use `variant="outline"` or `variant="ghost"` to make this less prominent.

Comment (SerpentBytes) [reply app/components/certificate/description.tsx:68]: If the goal is to make it less prominent, should we use `IconButton` and `Tooltip`?

Comment (humphd) [reply app/components/certificate/description.tsx:67]: I think having it be a button is good, just that we only have a single "Call to Action" button (i.e., refresh).

**Review by humphd — APPROVED** [review]

**Comment by humphd** [comment]
@SerpentBytes you still have this as "Draft."  Any particular reason?  I think we can merge.

**Pull request merged**